go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/fluxcd/image-automation-controller
 
-go 1.23.0
+go 1.24.0
+
+toolchain go1.24.1
 
```

Comment:
Please bump all packages including the /api and remove the toolchain hardcode. See my PR for Kube 1.33 and Go in SC.

Reply:
Fixed, thanks!